app/src/main/java/org/fossasia/phimpme/gallery/util/SecurityHelper.java

```diff
@@ -202,6 +202,7 @@ public void onClick(View view) {
                         String answer1 = "";
                         answer1 = SP.getString("Security Answer", answer1);
                         if (Objects.equals(answer1, securityAnswer1.getText().toString())) {
+                            dialog.dismiss();
                             changePassword(activity, passwordDialog);
```

Comment:
Move this above

Reply:
@abishekvashok can you please tell me what do I have to move above? I did not understand what is to be moved.

Reply:
Dismiss the dialog only after the changepassword function is called.